packages/context/src/resolver.ts

```diff
@@ -176,51 +183,34 @@ export function resolveInjectedArguments(
   // Example value:
   //   [ , 'key1', , 'key2']
   const injectedArgs = describeInjectedArguments(target, method);
-  nonInjectedArgs = nonInjectedArgs || [];
+  const extraArgs = nonInjectedArgs || [];
 
   const argLength = DecoratorFactory.getNumberOfParameters(target, method);
-  const args: BoundValue[] = new Array(argLength);
-  let asyncResolvers: Promise<void>[] | undefined = undefined;
 
   let nonInjectedIndex = 0;
-  for (let ix = 0; ix < argLength; ix++) {
-    const injection = ix < injectedArgs.length ? injectedArgs[ix] : undefined;
+  return resolveList(new Array(argLength), (v, i) => {
```

Comment:
Could you please preserve `ix` as the index variable/argument name? It will make the change-set smaller, which is always a good thing.